### PR TITLE
ja: Make docs/tasks/debug-application-cluster/debug-stateful-set.md follow v1.18 of the original text

### PR DESCRIPTION
**Pull request by shuzokato**
Updated docs/tasks/debug-application-cluster/debug-stateful-set.md to follow v1.18 of the original text. #23385

### CODE REVIEW

**Review by oke-py — APPROVED** [review]
/lgtm

File: content/ja/docs/tasks/debug-application-cluster/debug-stateful-set.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ StatefulSetに属し、ラベル`app=myapp`が設定されているすべてのP
 kubectl get pods -l app=myapp
 ```
 
-Podが長期間`Unknown`または`Terminating`の状態になっていることがわかった場合は、それらを処理する方法について[StatefulSet Podsの削除](/docs/tasks/manage-stateful-set/delete-pods/)タスクを参照してください。
+Podが長期間`Unknown`または`Terminating`の状態になっていることがわかった場合は、それらを処理する方法について[StatefulSet Podsの削除](/docs/tasks/run-application/delete-stateful-set/)タスクを参照してください。
```
Comment (inductor) [inline content/ja/docs/tasks/debug-application-cluster/debug-stateful-set.md:30]:
```suggestion
Podが長期間`Unknown`または`Terminating`の状態になっていることがわかった場合は、それらを処理する方法について[StatefulSet Podの強制削除](/ja/docs/tasks/run-application/delete-stateful-set/)タスクを参照してください。
```

ref. https://kubernetes.io/ja/docs/tasks/run-application/force-delete-stateful-set-pod/

**Review by inductor — CHANGES_REQUESTED** [review]

**Pull request merged**